cmd/tegola/cmd/root.go

```diff
@@ -89,23 +60,31 @@ func rootCmdValidatePersistent(cmd *cobra.Command, _ []string) (err error) {
 		build.Commands = append(build.Commands, cmdName)
 		return nil
 	default:
-		return initConfig(configFile, requireCache, logLevel, logger)
+		return initConfig(configFile, requireCache, logLevel)
 	}
 }
 
-func initConfig(configFile string, cacheRequired bool, logLevel string, logger string) (err error) {
-	err = validateSupportedLoggers(logger)
-	if err != nil {
-		return err
-	}
-	log.SetLogger(logger)
-
-	// set log level before the first log is called
-	level, err := getLogLevelFromString(logLevel)
-	if err != nil {
-		return err
-	}
-	log.SetLogLevel(level)
+func initConfig(configFile string, cacheRequired bool, logLevel string) (err error) {
+	// Parse the provided log level; default to INFO if parsing fails.
+	lvl := log.ParseLogLevel(logLevel)
```

Comment:
Note, `TRACE` doesn't exist in `slog` so the `log-level` usage needs to be updated.